openthread/cmake/extensions.cmake

```diff
@@ -0,0 +1,73 @@
+#
+# Copyright (c) 2020 Nordic Semiconductor ASA
+#
+# SPDX-License-Identifier: LicenseRef-BSD-5-Clause-Nordic
+#
+set(NRFXLIB_DIR ${ZEPHYR_NRFXLIB_MODULE_DIR})
+assert_exists(NRFXLIB_DIR)
+include(${NRFXLIB_DIR}/common.cmake)
+
+# Store the configuration of the compiled OpenThread libraries
+# and set source and destination paths.
+function(openthread_libs_configuration_write)
+  set(OPENTHREAD_SRC_DIR "${CMAKE_BINARY_DIR}/modules/openthread/build/src"
+    CACHE STRING "Directory containg the OpenThread source code.")
+
+  find_package(Git QUIET)
+  if(GIT_FOUND)
+    execute_process(
+      COMMAND           ${GIT_EXECUTABLE} rev-parse HEAD
+      WORKING_DIRECTORY ${OPENTHREAD_SRC_DIR}
```

Comment:
you haven't adressed this comment: https://github.com/nrfconnect/sdk-nrf/pull/2655#discussion_r487778529

Reply:
Used `ZEPHYR_OPENTHREAD_MODULE_DIR` instead.